src/silvimetric/resources/storage.py

```diff
@@ -165,7 +174,7 @@ def getMetrics(self) -> list[Metric]:
         return self.getConfig().metrics
 
     @contextlib.contextmanager
-    def open(self, mode:str='r', timestamp=None) -> tiledb.SparseArray:
+    def open(self, mode:str='r', timestamp=None) -> Generator[tiledb.SparseArray, None, None]:
```

Comment:
a Generator of what are you talking about?

Reply:
I'm not sure I understand, it's a generator of a tiledb array stream